cookiecutter.json

```diff
@@ -0,0 +1,17 @@
+{
+  "snippet_name": "MySnippet",
+  "project_name": "NetSquid-{{ cookiecutter.snippet_name }}",
+  "project_slug": "{{ cookiecutter.project_name|lower|replace(' ', '-') }}",
+  "project_folder": "{{ cookiecutter.project_slug|replace('-', '_') }}",
+  "description": "Write some description of the package",
+  "author_name": "Name LastName",
+  "author_email": "name@email.com",
+  "version": "0.0.0",
+  "license": "Apache-2.0",
+  "url": "https://github.com",
+  "docs_server": "docs.netsquid.org",
+  "pip_server": "pypi.netsquid.org",
+  "netsquid_min": "0.6.2",
+  "netsquid_max": "0.7.0",
```

Comment:
Rather than a min and max, something like a "version constraint" that is added as a suffix would be more flexible, so users have choice to only give min version, pin the version, give a range, or leave it empty for any netsquid, etc.

Reply:
Done